index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
+tbG1hV0ZpYkdWRGNtVmtaVzUwYVdGc0lpd2lVSEp2YjJaUFprRm5aVU55WldSbGJuUnBZV3dpWFN3aVk
+yeGhhVzBpT25zaVlXZGxUM1psY2lJNk1qRjlmUS5UUXltWms2Z0IyekRhUmVFNlVUYTlLQnA2MVRlX21
+0bzl3V1JDZDFDcWNkQWpOMFNZMTItVWhPbHN4em4tWGhaRE50R2RUd3pZX1ZVUklPR2JiQkROZ25HYlR
+zdHJ3bjgtU29oX3FpZzJsc3k3ZlM1ckF6QmdtZGFmSkxiYUtTQ0VsSGJ5RElsMVZ4NFFmYzk0LVdOVWJ
+pQ1RTaEp3dENJN204MmxMWjltSmdmIl19.SR-zmsg-i8n9WNj43zYX0LERb8UWfgYyHFLG3R2HmbhAS5
+mLFsy3xA1D8f6ksoT5UxOwr5T7uuj1UkC8v3aX1Z6yZus1HhhdhuhPnsMvkbBLJl7G6u4z8Fq7AJxT6e
+BhndD4Q6TfhcWXisrZaR1Al1921L_CfOqq5ZD1C1EdaoM
+          </pre>
+
+            <p>The example above contains the following JWT header and payload.</p>
+            <pre class="example nohighlight" title="Presentation JWT header">
+{
+  "alg": "RS256",
+  "typ": "JWT"
+}
+          </pre>
+
+            <pre class="example nohighlight" title="Presentation JWT payload">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "jti": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+  "iss": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "aud": "https://dmv.example.gov/issuers/14",
+  "nonce": "343§$FSFDa-",
+  "iat": "1541493724",
+  "exp": "1573029723",
+  "type": ["VerifiablePresentation"],
+  "verifiableCredential": ["
+eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJAY29udGV4dCI6WyJodHRwczovL3czLm9yZy8yMDE
+4L2NyZWRlbnRpYWxzL3YxIiwiaHR0cHM6Ly9leGFtcGxlLmNvbS9leGFtcGxlcy92MSJdLCJqdGkiOiJ
+1cm46dXVpZDozOTc4MzQ0Zi04NTk2LTRjM2EtYTk3OC04ZmNhYmEzOTAzYzUiLCJzdWIiOiJkaWQ6ZXh
+hbXBsZTplYmZlYjFmNzEyZWJjNmYxYzI3NmUxMmVjMjEiLCJhdWQiOiJodHRwczovL2Rtdi5leGFtcGx
+lLmdvdi9pc3N1ZXJzLzE0Iiwibm9uY2UiOiIzNDPCpyRGU0ZEYS0iLCJpYXQiOiIxNTQxNDkzNzI0Iiw
+iZXhwIjoiMTU3MzAyOTcyMyIsInR5cGUiOlsiVmVyaWZpYWJsZVByZXNlbnRhdGlvbiJdLCJ2ZXJpZml
+hYmxlQ3JlZGVudGlhbCI6WyJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKQVk
+yOXVkR1Y0ZENJNld5Sm9kSFJ3Y3pvdkwzY3pMbTl5Wnk4eU1ERTRMMk55WldSbGJuUnBZV3h6TDNZeEl
+pd2lhSFIwY0hNNkx5OWxlR0Z0Y0d4bExtTnZiUzlsZUdGdGNHeGxjeTkyTVNKZExDSnpkV0lpT2lKa2F
+XUTZaWGhoYlhCc1pUcGxZbVpsWWpGbU56RXlaV0pqTm1ZeFl6STNObVV4TW1Wak1qRWlMQ0pxZEdraU9
+pSm9kSFJ3T2k4dlpHMTJMbVY0WVcxd2JHVXVaMjkyTDJOeVpXUmxiblJwWVd4ekx6TTNNeklpTENKcGM
+zTWlPaUpvZEhSd2N6b3ZMMlJ0ZGk1bGVHRnRjR3hsTG1kdmRpOXBjM04xWlhKekx6RTBJaXdpYVdGMEl
+qb2lNVFUwTVRRNU16Y3lOQ0lzSW1WNGNDSTZJakUxTnpNd01qazNNak1pTENKMGVYQmxJanBiSWxabGN
+tbG1hV0ZpYkdWRGNtVmtaVzUwYVdGc0lpd2lVSEp2YjJaUFprRm5aVU55WldSbGJuUnBZV3dpWFN3aVk
+yeGhhVzBpT25zaVlXZGxUM1psY2lJNk1qRjlmUS5UUXltWms2Z0IyekRhUmVFNlVUYTlLQnA2MVRlX21
+0bzl3V1JDZDFDcWNkQWpOMFNZMTItVWhPbHN4em4tWGhaRE50R2RUd3pZX1ZVUklPR2JiQkROZ25HYlR
+zdHJ3bjgtU29oX3FpZzJsc3k3ZlM1ckF6QmdtZGFmSkxiYUtTQ0VsSGJ5RElsMVZ4NFFmYzk0LVdOVWJ
+pQ1RTaEp3dENJN204MmxMWjltSmdmIl19.EjTFSFneWu2duLhy8DE1to943Shl_u70sSy8BSLBLlr10K
+Q1dzyb-L8P14luAd9Jk-8bNkqI5XuqJiFF2mgZuEwYugh98U0I5d7g2jlOqeXWHMAYaKEELfUsiNn5Bj
+AqEp58JEN7e-pSWYHCX49BsQmQumDsnDFBDzIHoTQe3no"]
+}
+          </pre>
+
+            <p class="issue">
```

Comment:
I'm supportive of pulling this PR in as long as:

1. There are multiple implementers - who are the multiple implementers for this PR?
2. We are very clear about the limitations of VCs encoded as JWTs - signatures are tied to a very specific serialization where whitespace destroys the signature, signatures are tied to JSON, base64 encoding makes developers have to use twice as much storage in the simples case, set and chained signatures are problematic, no support for other types of mathematical proof formats, if JSON-LD processing is not done there is no guarantee that the semantics of the message is well formed, etc. We need a very large warning section here so that developers know what they're getting themselves into when using JWTs.

Digital Bazaar's position is that combining VCs with JWTs are a footgun, we won't be implementing it, but we also don't want to stand in the way of other companies that still want to go down that path in spite of all of the warnings.